docs/orchestrating-elastic-stack-applications/elasticsearch/node-configuration.asciidoc

```diff
@@ -10,6 +10,8 @@ endif::[]
 
 Any setting defined in the `elasticsearch.yml` configuration file can also be defined for a set of Elasticsearch nodes in the `spec.nodeSets[?].config` section.
 
+Some settings cannot be changed and should not be changed, check <<{p}-reserved-settings>>.
```

Comment:
nit: I don't think we reject a manifest in which settings managed by the operator are changed. So, technically speaking, they can be changed.

```suggestion
Some settings are managed by ECK, it is not recommended to change them, refer to <<{p}-reserved-settings>> for more details.
```

Reply:
I'm also wondering if we should not prefix this with `NOTE:`